examples/dt/uni04delta/edpm.md

```diff
@@ -0,0 +1,64 @@
+# Deploying the OpenStack dataplane
```

Comment:
This should be renamed to "deploying compute nodes".

In the steps in `edpm-ceph.md` are you deploying dataplane nodes which host Ceph.
In the steps in `edpm.md` you are deploying dataplane nodes which host Nova compute.

Reply:
Fixed